### PR TITLE
Changelog 4.5.1

### PR DESCRIPTION
**Pull request by directionless**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,71 @@
 # osquery Changelog
 
+<a name="4.5.1"></a>
+## [4.5.1](https://github.com/osquery/osquery/releases/tag/4.5.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.5.0...4.5.1)
+
+
+### New Features
```
Comment (muffins) [inline CHANGELOG.md:9]: `New Features` could maybe be a level 2 header as it seems the level 3s below are all associated with it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,71 @@
...
+
+### Under the Hood improvements
+
+- carver: Improve tests by faking postCarve ([#6659](https://github.com/osquery/osquery/pull/6659))
+- carver: Emit an error if the carve function is disabled ([#6658](https://github.com/osquery/osquery/pull/6658))
+- carver: Update carves specs to allow full scan ([#6657](https://github.com/osquery/osquery/pull/6657))
```
Comment (muffins) [inline CHANGELOG.md:15]: The wording here seems a little off, maybe `Update carver specs to allow full scan`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,71 @@
...
+- carver: Update table to use JSON ([#6656](https://github.com/osquery/osquery/pull/6656))
+- Improve performance and accuracy of Windows registry querying ([#6647](https://github.com/osquery/osquery/pull/6647))
```
Comment (muffins) [inline CHANGELOG.md:17]: The other line items all seem to have `<category>:` prefixes, perhaps this could use `performance:`?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,71 @@
...
+- refactor: Move ephemeral database plugin into core and simplify tests ([#6648](https://github.com/osquery/osquery/pull/6648))
+
+### Table Changes
+
+- Support for Office MRU (most recently used) entries ([#6587](https://github.com/osquery/osquery/pull/6587))
+- Implement configurable timeout through WHERE clause on `curl_certificate` ([#6641](https://github.com/osquery/osquery/pull/6641))
+- Add atom table spec to window ([#6649](https://github.com/osquery/osquery/pull/6649))
```
Comment (muffins) [inline CHANGELOG.md:24]: Maybe atom here could be capitalized to better indicate it's the editor being referred to?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,12 +1,71 @@
...
+- Add signature information to authenticode table on windows ([#6677](https://github.com/osquery/osquery/pull/6677))
+
+### Bug Fixes
+
+- Fix container overflow in curl_certificate ([#6664](https://github.com/osquery/osquery/pull/6664))
+- Fix handling of invalid array bound error with EvtNext function ([#6660](https://github.com/osquery/osquery/pull/6660))
+- Fix BIOS Info table searching. ([#5246](https://github.com/osquery/osquery/pull/5246))
```
Comment (muffins) [inline CHANGELOG.md:31]: This line has a period whereas the rest do not, maybe remove for consistency?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,12 +1,71 @@
...
+- Fix image column within drivers table on Windows ([#6652](https://github.com/osquery/osquery/pull/6652))
+- Fix windows dirPathsAreEqual to use the documented way ([#6690](https://github.com/osquery/osquery/pull/6690))
+- Add Addition AWS regions ([#6666](https://github.com/osquery/osquery/pull/6666))
```
Comment (muffins) [inline CHANGELOG.md:34]: This looks like it might actually go under the `Table Changes` section as it doesn't look to be a bug.

**Review by muffins — COMMENTED** [review]
Overall lgtm! Thanks a lot for putting this together, I had a couple of notes/comments/questions for you if you don't mind.

**Comment by theopolis** [comment]
Ok, the final few fixes have been merged. This PR is the next step towards a 4.5.1.

File: CHANGELOG.md (hunk 1 above)
Comment (directionless) [reply CHANGELOG.md:9]: As it stands, All subheadings are 3 deep, and nested under the `4.5.1` tag. Previously, we had `## New Features / Under the Hood improvements` but it felt better split. There are no items in `New Features`

I suspect it looks more sensible in the whole view. and not the diff. But maybe there's something more clear

File: CHANGELOG.md (hunk 2 above)
Comment (directionless) [reply CHANGELOG.md:15]: WFM

File: CHANGELOG.md (hunk 3 above)
Comment (directionless) [reply CHANGELOG.md:17]: I think the prefixes are an anti pattern. I'll remove them all

File: CHANGELOG.md (hunk 6 above)
Comment (directionless) [reply CHANGELOG.md:34]: Seemed like a toss up. It's adding additional regions, but not really changing anything. Moved it

**Review by theopolis — APPROVED** [review]

**Pull request merged**